README.md

```diff
@@ -7,25 +7,30 @@ A proof of concept / template for using Wasmoon and Teal to make a game that run
 - [npm](https://www.npmjs.com/)
 - [lua](https://www.lua.org/)
 - [luarocks](https://luarocks.org/)
+
+### Lua requirements (installed using luarocks)
 - [teal](https://github.com/teal-language/tl)
+- [cyan](https://github.com/teal-language/cyan)
```

Comment:
I've not had the time to fully review this yet, but just glancing at this: do we really need all of these deps? I see the JS is using things like Babel and Webpack to bundle a really tiny static website. Could we, out of simplicity, drop some of these deps?

Same with `cyan`, is it _really_ needed?

Obviously this is not the PR to address this but to start the conversation.

Reply:
We probably could simplify the javascript side of things. As for `cyan`, I'm temporarily removing this dependency, as I can't seem to get the latest teal working for this project. But in theory, we would need it to compile the teal down to lua - unless we injected the teal runtime into the app.

It'd be a nice extra issue to create, to see how much we can streamline this.

Reply:
Ah I see, I thought it was just some build system for automating things, not that it was crucial for the teal to lua steps.